uwblueprint/social-diversity-for-children
# feat(session sensitive sidebar): component tweak

### Implementation description

Admin Navbar: 
![image](https://user-images.githubusercontent.com/44826218/142781121-20ec3768-ba1f-40f3-b691-33480240a693.png)

Teacher Navbar:
![image](https://user-images.githubusercontent.com/44826218/142781138-662e0471-9f1b-4ebc-80e2-77387de60cfd.png)

- Add Hide admin-only links from sidebar depending on user session



### Steps to test

1. Visit all internal page with both admin and volunteer, ensure that the corresponding navbar shows.



### What should reviewers focus on?

- Design: new icon and link name for browse internal user page on sidebar
- Design: Look and feel of sidebars depending on context of sidebar

### Checklist
